locales/zh-cn.json

```
@@ -43,6 +43,8 @@
   "config.include_subfolders": "搜索子目录",
   "config.indent": "文案文件的缩进大小",
   "config.keep_fulfill": "始终用空字符串填充所有的键",
+  "config.key_prefix": "文案 key 的前缀",
```

```suggestion
  "config.key_prefix": "提取出的文案路径前缀",
```
Let's use `路径` here to better match with other translations.